src/App.tsx

```diff
@@ -16,35 +17,90 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [goodsList, setGoodsList] = useState<string[]>([...goodsFromServer]);
+  const [sortType, setSortType] = useState<string>('');
+  const [reversed, setReversed] = useState<boolean>(false);
```

Comment:
we don't need to specify types for primitives 
```suggestion
  const [sortType, setSortType] = useState('');
  const [reversed, setReversed] = useState(false);
```